cmd/gmailctl/cmd/root_cmd.go

```diff
@@ -51,7 +51,7 @@ func init() {
 	// Here you will define your flags and configuration settings.
 	// Cobra supports persistent flags, which, if defined here,
 	// will be global for your application.
-	rootCmd.PersistentFlags().StringVar(&cfgDir, "config", "", "config directory (default is $HOME/.gmailctl)")
+	rootCmd.PersistentFlags().StringVar(&cfgDir, "config", "", "config directory (default is $HOME/.config/gmailctl unless $HOME/.gmailctl is already present)")
```

Comment:
The wording is 100% correct, but might be unnecessary detail to newcomers. I would perhaps word it as:

default search paths: $HOME/.gmailctl, $HOME/.config/gmailctl

Reply:
Sounds good. Do you want to indicate that the latter will be created by default, or just leave implicit?

Reply:
```suggestion
	rootCmd.PersistentFlags().StringVar(&cfgDir, "config", "", "config directory (default search paths: $HOME/.gmailctl, $HOME/.config/gmailctl)")
```